app/app.go

```diff
@@ -433,7 +433,7 @@ func NewApp(logger log.Logger, db dbm.DB, traceStore io.Writer, appOpts AppOptio
 	app.mm.SetOrderBeginBlockers(
 		upgrade.ModuleName, mint.ModuleName, distr.ModuleName, slashing.ModuleName,
 		validatorvesting.ModuleName, kavadist.ModuleName, auction.ModuleName, cdp.ModuleName,
-		bep3.ModuleName, hard.ModuleName, committee.ModuleName, issuance.ModuleName, incentive.ModuleName,
+		bep3.ModuleName, committee.ModuleName, hard.ModuleName, issuance.ModuleName, incentive.ModuleName,
```

Comment:
Along similar lines - move `committee` in front of `cdp` begin blocker so that interest accumulation reflects any changes from proposals